oauth2/handler.go

```diff
@@ -225,6 +225,7 @@ func (h *Handler) WellKnownHandler(w http.ResponseWriter, r *http.Request) {
 		TokenURL:                           urlx.AppendPaths(h.c.IssuerURL(), TokenPath).String(),
 		JWKsURI:                            urlx.AppendPaths(h.c.IssuerURL(), JWKPath).String(),
 		RevocationEndpoint:                 urlx.AppendPaths(h.c.IssuerURL(), RevocationPath).String(),
+		IntrospectionEndpoint:              urlx.AppendPaths(h.c.IssuerURL(), IntrospectPath).String(),
```

Comment:
Introspection is running on the Admin endpoint and would thus be incorrectly referenced here.

Reply:
Okay, thanks for the review! That's fine with me if we reference the Admin endpoint.

I don't see a variable for the Admin endpoint anywhere in this file, nor can I find `h.c.AdminURL()` or any related-looking term. `OAuth2ClientRegistrationURL()` is about the closest that I see.

I imagine that if I can find the Admin endpoint, we would use that instead of `h.c.IssuerURL()`. Could you please direct me to where I could find that?

Reply:
Yeah that doesn't exist because it wasn't needed until now. Why don't we create - similar to e.g. `h.c.OIDCDiscoverySupportedClaims(),` a new config value that, if set, sets this key?